libr/core/linux_heap_glibc.c

```diff
@@ -849,63 +853,123 @@ void GH(print_heap_fastbin)(RCore *core, GHT m_arena, MallocState *main_arena, G
 	}
 }
 
-static void GH(print_tcache_instance)(RCore *core, GHT m_arena, MallocState *main_arena) {
-	if (!core || !core->dbg || !core->dbg->maps) {
-		return;
+static GH(RTcache)* GH(tcache_new) (tcache_type type) {
+	GH(RTcache) *tcache = R_NEW0 (GH(RTcache));
+	tcache->type = type;
+	if (tcache->type == NEW) {
+		tcache->RHeapTcache.heap_tcache = R_NEW0(GH(RHeapTcache));
+	} else {
+		tcache->RHeapTcache.heap_tcache_old = R_NEW0(GH(RHeapTcacheOld));
+	}
+	return tcache;
+}
+
+static void GH(tcache_free) (GH(RTcache)* tcache) {
+	free(tcache);
+}
+
+static bool GH(tcache_read) (RCore *core, GHT tcache_start, GH(RTcache)* tcache) {
+	bool r = false;
```

Comment:
return with ternary operator, but i dont see which is the difference between the true and false blocks

Reply:
right!